lib/shelley/cardano-wallet.cabal

```diff
@@ -61,6 +61,7 @@ library
     , filepath
     , fmt
     , generic-lens
+    , int-cast
```

Comment:
We use a bewildering variety of packages. At some point, we may want to document and consolidate our choices, so that writing mundane code becomes more streamlined (e.g. Text vs ByteString vs lazy ByteString; converting to/from hex). 😅 But I do not believe that the variety really hurts readability, so the status quo is fine as well.

Reply:
I do agree we use a very large variety of packages!

Though to some extent, perhaps this reflects the fragmentation and/or modularity within the Haskell package ecosystem?

In this case, I believe the `int-cast` package is rather well-designed and very useful for catching errors. We actually already do use it within `cardano-wallet`: this just adds it to some of the other build targets.

Reply:
> ByteString vs lazy ByteString

To some extent, I think our hand is forced here by the packages we rely on.

Historically, Aeson has used lazy `ByteString` by default. Though I've noticed that strict variants have recently been added for the most common functions.